src/components/Navbar/Navbar.jsx

```diff
@@ -26,6 +48,14 @@ const Navbar = ({ isAdmin, changesMade }) => {
     { text: 'Emergency Numbers', path: '/emergency-numbers' },
   ];
 
+  const togglePortal = () => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'togglePortal' is assigned a value but never used.